README.md

```diff
@@ -195,6 +197,31 @@ self.onmessage = async (event) => {
 
 </details>
 
+<details>
+    <summary>
+        <b>Browser</b>
+    </summary>
+
+```html
+<script type="text/javascript" src="https://unpkg.com/canvg@3.0.0-beta.0/lib/umd.js"></script>
+<script type="text/javascript">
+window.onload = async () => {
+
+    const canvas = document.querySelector('canvas');
+    const ctx = canvas.getContext('2d');
+    
+    v = await canvg.Canvg.fromString(ctx, '<svg width="600" height="600"><text x="50" y="50">Hello World!</text></svg>');
```

Comment:
`fromString` is "sync" method, no `await` needed.

Reply:
fixed! updated jsfiddle as well